examples/chef/common/stubs.cpp

```diff
@@ -235,3 +235,7 @@ void emberAfWakeOnLanClusterInitCallback(EndpointId endpoint)
     WakeOnLan::SetDefaultDelegate(endpoint, &wakeOnLanManager);
 }
 #endif
+
+// No-op function, used to force linking this file,
+// instead of the weak functions from other files
+extern "C" void chef_include_stubs_impl(void) {}
```

Comment:
Is this only available to ESP32?

Reply:
Yes, if other platforms have the same issues they can also use this functions